ideascube/monitoring/templates/monitoring/specimen_confirm_delete.html

```diff
@@ -1,4 +1,4 @@
 {% extends 'ideascube/form_confirm_delete.html' %}
 {% load i18n %}
 
-{% block confirm_message %}{% blocktrans with specimen=specimen.barcode item=specimen.item %}Are you sure you want to delete the specimen "{{ specimen }}" of "{{ item }}"?{% endblocktrans %}{% endblock %}
+{% block confirm_message %}{% blocktrans with specimen=specimen.pretty_identifier item=specimen.item %}Are you sure you want to delete the specimen "{{ specimen }}" of "{{ item }}"?{% endblocktrans %}{% endblock %}
```

Comment:
Indeed, the fact it is a sentence makes it more tricky.

But let's take a step back.

Currently, if someone creates a specimen without a barcode, then when trying to delete it they would see:

![screenshot from 2017-01-12 18-18-57](https://cloud.githubusercontent.com/assets/272170/21900322/aab61714-d8f3-11e6-8386-0e1a74f75091.png)

That is extremely unhelpful, to say the least. Basically, that sentence is just as useful as saying:

> Are you sure you want to delete this specimen of "Blah" ?

So I guess we could just say this. 😄 

But then, in the case where there actually is a barcode for the specimen, showing it to the user can reassure them that they didn't misclick. It can also be useful for someone who clicks on the delete link, gets called by someone else, then comes back to that confirmation dialog.

So it is interesting to keep it, maybe.

Now, it is entirely possible to create a specimen with neither a `barcode` nor a `serial`. This surprises me quite a lot to be honest, but that's how the code currently works. Which means that even with the above change, that sentence could still read as unhelpful as above in some cases.

I think this confirmation dialog needs to be redone:

* it needs a cancel/no button
* the title should be something more generic like "Are you sure you want to delete this specimen?"
* the details should be displayed separately, and we should display all the details which can help the user confirm: barcode, serial, quantity, whether it has an associated file, stockitem name, module, description, ...

I also think this is a different change from this one (I'll be happy to open a ticket if you want me to).

Therefore, I'd argue for you to leave this sentence unchanged from master.